pkg/client/clientset/versioned/clientset.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2020 The Kubernetes Authors.
+Copyright 2021 The Kubernetes Authors.
```

Comment:
Since this files were first generated in 2020, should this not be 2020?
I'm not sure how generated code and copyright works.

Reply:
I don't know; it's just what it does. We need this for all the other PRs that are failing CI :)

Reply:
I know. I'm just not sure about the legal concerns here. It shouldn't be a problem since this is generated code, but I'm not sure. @robscott?

Reply:
The code generator regenerated code; it's just that there were no textual differences. There's no fix to this other than to change codegen to do some date magic based on existing output files and I don't see that happening.

Reply:
This is really strange. I haven't seen run into this on k/k but I'm not really sure how it's avoided. Can't find anything in that [codegen script](https://github.com/kubernetes/kubernetes/blob/master/hack/update-codegen.sh) that would workaround this. As I've dug a bit further, it seems like many subprojects simply omit the date from the header. Maybe that would make the most sense here?

* https://github.com/kubernetes-sigs/cluster-api/blob/master/hack/boilerplate.go.txt
* https://github.com/kubernetes/sample-controller/blob/master/hack/boilerplate.go.txt
* https://github.com/kubernetes/sample-apiserver/blob/master/hack/boilerplate.go.txt
* https://github.com/kubernetes/code-generator/blob/master/hack/boilerplate.go.txt

Reply:
happy to file a separate PR for that